deploy/monitoring_reader_role.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: muo-monitoring-reader
+  namespace: openshift-monitoring
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
+  - secrets
+  - service
```

```suggestion
  - services
```

Since PR #273 was merged, this needs to be `services` or else the following error is produced:

```
services "prometheus-k8s" is forbidden: User "system:serviceaccount:openshift-managed-upgrade-o
perator:managed-upgrade-operator" cannot get resource "services" in API group "" in the namespace "openshift-monitoring"
```